papergres.go

```diff
@@ -724,10 +742,10 @@ func (r *Result) setMeta(m meta) {
 	r.LastInsertId.ID = m.LastInsertId
 	r.RowsAffected.Count = m.RowsAffected
 	if m.LastInsertId == 0 {
-		r.LastInsertId.Err = errors.New("No LastInsertId returned")
+		r.LastInsertId.Err = errors.New("no LastInsertId returned")
 	}
 	if m.RowsAffected == -1 {
-		r.RowsAffected.Err = errors.New("No RowsAffected returned")
+		r.RowsAffected.Err = errors.New("no RowsAffected returned")
```

Comment:
Why change the case?

Reply:
Error strings shouldn't be capitalized according to Go standards. https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Yes and according to the doc the reason for that is so logging "formats without a spurious capital letter mid-message", yet there are caps mid message in this error message.